scripts/build.js

```
@@ -0,0 +1,85 @@
+);
+
+// prepare the dts
+let dts = fs.readFileSync(`./temp/node/float.d.ts`, {
+  encoding: "utf-8",
+});
+dts = dts.replace(
+  `/* tslint:disable */
+/* eslint-disable */`,
+  ""
+);
+dts = "/* this file is auto-generated, do not modify */\n" + dts;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)